internal/dinosaur/pkg/clusters/ocm_provider.go

```diff
@@ -359,7 +360,11 @@ func (o *OCMProvider) createSyncSet(clusterID string, resourceSet types.Resource
 		return nil, errors.WithStack(sysnsetBuilderErr)
 	}
 
-	return o.ocmClient.CreateSyncSet(clusterID, syncset)
+	s, err := o.ocmClient.CreateSyncSet(clusterID, syncset)
+	if err != nil {
+		return nil, fmt.Errorf("creating SyncSet: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("creating SyncSet for cluster %q: %w", clusterID, err)
```

I don't know what spelling for sync set would be best (SyncSet, syncset, Syncset, Sync Set, sync set, sync-set, etc.), but I would suggest to use the same spelling in all error wrappings/log messages.

Reply:
I think it's an OpenShift resource and it's spelt SyncSet